test/e2e/nstemplatetier_test.go

```diff
@@ -153,54 +152,50 @@ func TestUpdateNSTemplateTier(t *testing.T) {
 	hostAwait := awaitilities.Host()
 	memberAwait := awaitilities.Member1()
 
-	// first group of users: the "cheesecake lovers"
-	cheesecakeSyncIndexes := setupAccounts(t, awaitilities, "cheesecake", "cheesecakelover%02d", memberAwait, count)
-	// second group of users: the "cookie lovers"
-	cookieSyncIndexes := setupAccounts(t, awaitilities, "cookie", "cookielover%02d", memberAwait, count)
-
-	// setup chocolate tier to be used for creating spaces without usersignups
-	spaces := setupSpaces(t, awaitilities, "chocolate", "chocolatelover%02d", memberAwait, count)
-
-	cheesecakeSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cheesecakeSyncIndexes, "cheesecake", "base", "base", true)
-	cookieSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cookieSyncIndexes, "cookie", "base", "base", true)
-	verifyResourceUpdatesForSpaces(t, awaitilities, memberAwait, spaces, "chocolate", "base", "base")
-
-	// when updating the "cheesecakeTier" tier with the "advanced" template refs for namespaces (ie, same number of namespaces) but keep the ClusterResources refs
-	updateTemplateTier(t, hostAwait, "cheesecake", "advanced", "")
-	// and when updating the "cookie" tier with the "baseextendedidling" template refs (ie, same number of namespaces)
-	updateTemplateTier(t, hostAwait, "cookie", "baseextendedidling", "baseextendedidling")
-	// and when updating the "chocolate" tier to the "advanced" template for Namespace refs
-	updateTemplateTier(t, hostAwait, "chocolate", "advanced", "")
-
-	// then
-	cheesecakeSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cheesecakeSyncIndexes, "cheesecake", "advanced", "base", false)
-	cookieSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cookieSyncIndexes, "cookie", "baseextendedidling", "baseextendedidling", false)
-	verifyResourceUpdatesForSpaces(t, awaitilities, memberAwait, spaces, "chocolate", "advanced", "base")
+	WaitUntilBaseNSTemplateTierIsUpdated(t, hostAwait)
+	baseTier, err := hostAwait.WaitForNSTemplateTier("base")
+	require.NoError(t, err)
 
-	// when updating the "cheesecakeTier" tier with the "advanced" template refs for ClusterResources but keep the Namespaces refs
-	updateTemplateTier(t, hostAwait, "cheesecake", "", "advanced")
-	// and when updating the "cookie" tier to the "base" template refs
-	updateTemplateTier(t, hostAwait, "cookie", "base", "base")
-	// and when updating the "chocolate" tier to the "advanced" template for Namespace refs
-	updateTemplateTier(t, hostAwait, "chocolate", "", "advanced")
```

Comment:
I noticed that you dropped the second batch of updates, but I haven't found anything that wouldn't be already covered in the first step. Could you please confirm that?

Reply:
yes, that was my feeling as well. But we can keep this second batch of update/test if it's really worth it.